courses/free-courses-en.md

```diff
@@ -942,6 +943,13 @@
 * [Shaping up with Angular.js](https://www.codeschool.com/courses/shaping-up-with-angular-js) - Codeschool (Codeschool account *required*)
 
 
+#### Bun.js
+
+* [Bun Js - For Beginners (Playlist)](https://www.youtube.com/watch?v=3S2In1X07G8&list=PLT8tjCYKynKsXBrCf7KiQ6Y-6U304DjXT) - TechWebDocs
```

Comment:
We prefer playlist links, please provide one instead of a single video link.

Reply:
> We prefer playlist links, please provide one instead of a single video link.

thank you for you review, I provided one with the playlist link and other two were like crash courses for people who just want a fast recap of topics, shall I edit them out?